Buildings/Fluid/AirFilters/Empirical.mo

```diff
@@ -0,0 +1,313 @@
+        Ellipse(
+          extent={{-54,70},{-48,64}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-62,46},{-56,40}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-50,36},{-44,30}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-64,24},{-58,18}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-62,-2},{-56,-8}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-46,-12},{-40,-18}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-64,-26},{-58,-32}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-48,-30},{-42,-36}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-60,-52},{-54,-58}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{38,38},{44,32}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{40,-16},{46,-22}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Text(
+          extent={{-145,-100},{155,-140}},
+          textColor={0,0,255},
+          textString="%name")}),
+Diagram(coordinateSystem(preserveAspectRatio=false)),
+Documentation(info="<html>
+<p>
+An empirical model of air filters, which considers the impacts of the contaminant
+accumulation on the pressure drop and the filtration efficiency.
+The characteristics of the filters are defined in the performance dataset
+<code>per</code> that determines:
+</p>
+<ul>
+<li>
+the maximum amount of contaminants <code>mCon_nominal</code> that
+the filter can capture before reaching its nominal capacity;
+</li>
+<li>
+the types of contaminants can be captured by the filter;
+</li>
+<li>
+how the flow coefficient changes as contaminants build up;
+</li>
+<li>
+how the filtration efficiency changes along with the contaminant accumulation,
+specific to each type.
```

Comment:
each type of what? is it each type of the contaminant? I wonder if this bullet refers to the `per.filEffPar`?

Reply:
each type of  containment, yes, it refers to <code>per.filEffPar</code>. I will improve the whole document section to avoid confusion.